tests/test-td-payload/src/test.json

```diff
@@ -34,5 +37,29 @@
         },
         "result": "None",
         "run": false 
-    }        
-}
\ No newline at end of file
+    },
+    "tcs004": {
+        "name": "tdacpi001",
+        "hob": 0,
```

Comment:
why we need add hob with all zero data?

Reply:
The function build_testcases (which is at tests/test-td-payloader/src/main.rs) parse the test.json and generate the TestCases object. TestCases contains the TdAcpi object which is parsed from test.json. TdAcpi need the information of hob so that it can parse hob to get the ACPI tables. The actual value of hob is passed to TdAcpi before it is started to run.
In test.json hob field is a place holder so that build_testcases can generate the TdAcpi object.

Reply:
The field of "hob" means the base address of hob list. The name is confusing. It has been updated to hob_address in the new version.

Reply:
But I don't understand why we need input hob_address in jason file.

Why does a test case writer need aware there is hob_address field and need initialize to zero?

Reply:
Not all the test case need a hob_address field. For example Hob address is not needed in Testcase of TdInfo. 
But there is some testcase need be aware of hob_address. TdAcpi is the example. Because ACPI tables are passed in hob_list and the Testcase of TdAcpi parses hob_list to check if the tables exist and if the tables are correct.
So in the new version of test.json only tcs004 has the field of hob_address.

test.json describes the test cases. But hob_address is unknown to the test.json. So the initial value of hob_address is set to 0. The actual value will be set when TdAcpi is added to testsuites.

Reply:
> But hob_address is unknown to the test.json.

That is exactly my point. If hob_address is unknown to a case developer, what is the value to set to 0 in test.json?
It should be totally invisible to case developer.

Reply:
It's the test framework limitation, json data should mapping with test case structure. So if min want to use hob, he need add hob_address in json file. Then json data can be parsed successfully.